adam/flask_backend/app.py

```diff
@@ -110,7 +118,7 @@ def get_scene() -> Any:
         "test_curriculum": testing_curriculum,
         "scene_number": scene_number,
         "scene_images": [
-            path.rsplit("data", maxsplit=1)[-1]
+            path.rsplit("data/", maxsplit=1)[-1]
```

Comment:
Currently we split on the word `data` being present, but that was posing some problems for me (it left the `/` after data which made python interpret resulting paths as absolute). However, this could pose an issue if either the train or test curriculum directories contain `data` as part of the name.

I think we could circumvent this with two approaches

1. Only return the file name. Since we have the situation number, training path, and testing path already present in the front end, we can reconstruct the file path.
2. Return the object image information directly with `get_image` here, rather than the image path

Reply:
So one [could also use](https://docs.python.org/3.7/library/os.path.html#os.path.relpath) `os.path.relpath(path, str(DATA_DIR))`. If I'm reading correctly `experiment_dir` should be an absolute path and so should the glob results, so this should work.

Reply:
That does work!